### PR TITLE
How to Build a Deep Learning Model Using Tensorflow with Python

### PR DESCRIPTION
**Pull request by TimedH**
Here is the PR on how to build a deep learning model using tensorflow with python, i believe it will be of good help to readers.

### CODE REVIEW

**Comment by TimedH** [comment]
Hello @Kayere kindly be my peer reviewer in this PR.

**Comment by TimedH** [comment]
This closes #4194

**Comment by WanjaMIKE** [comment]
Plagiarism check is done. @lalith1403 just an alert, there seems to be duplicate files in this PR. Thanks

**Comment by lalith1403** [comment]
@TimedH can you delete the duplicate file? Thank you

**Comment by TimedH** [comment]
Hi @lalith1403 changes requested are in place

File: how-to-build-deep-learning-model-using-tensorflow-with-python/Index.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+#### __What is Tensorflow?__
+TensorFlow <span style="color:green">is a free, open-source machine learning software that can be used from start to end.
+</span> It has a large, flexible set of operators, libraries, and community resources that allow academics to advance the state-of-the-art in machine learning and developers to quickly construct and deploy machine learning applications.
+Tensorflow was developed in 2015 by google.
+
+#### **Merits and Demerits of TensorFlow**
+The following are some of the advantages of tensorflow ;
+
+#### Merits
+- Tensorflow is very compatible with a lot of languages like;  javascript, python, c#, swift, c++, and ruby hence the user may be choose any language to use tensorflow.
```
Comment (lalith1403) [inline how-to-build-deep-learning-model-using-tensorflow-with-python/Index.md:38]: Languages are proper nouns...

File: how-to-build-deep-learning-model-using-tensorflow-with-python/Index.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+- With its graphical approach, TensorFlow offers a superior way of visualizing data hence with the aid of TensorBoard, it is possible to easily debug nodes.
```
Comment (lalith1403) [inline how-to-build-deep-learning-model-using-tensorflow-with-python/Index.md:39]: This is factually incorrect. The underlying structure is a graph. It is not the approach.

File: how-to-build-deep-learning-model-using-tensorflow-with-python/Index.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+- It offers graphical support in deep learning by 
+allowing you to create neural networks using graphs that represent operations as nodes.
+- It is compatible with Keras, allowing users to program several high-level functionality areas. 
+- TensorFlow also has its own TPU architecture, which makes calculations faster than the GPU and CPU. Models built using TPU can be easily installed in the cloud at a cheaper price and used at a faster price.
```
Comment (lalith1403) [inline how-to-build-deep-learning-model-using-tensorflow-with-python/Index.md:43]: Factually incorrect.

**Review by lalith1403 — CHANGES_REQUESTED** [review]
There are many changes required in this article. Please consider making the necessary changes. Thank you. I have highlighted only a few of the errors in the article. There are several errors.

**Comment by hectorkambow** [comment]
@TimedH 
Good afternoon and thank you for submitting your topic suggestion.
 It seems this PR is still quite a bit aways from being completed. 
 
As a rule of thumb - please only submit articles (pull requests) with a complete and fully polished article - ready to be published. 
 
All revisions and edits should be completed on your own forked repo (as to not take up room in the queue). 
 
**We will be closing this and please re-submit when it is ready to be published as a new PR** (and you can add the duplicate label) and ping the reviewer and myself. 

Closing a PR does NOT mean the article can not be published (or fixed) it simply means that the edits, revisions, and fixes will happen outside of the queue.
 
👍 

cc @lalith1403

**Pull request closed without merging**